src/test/java/at/ac/tuwien/kr/alpha/AlphaTest.java

```diff
@@ -126,6 +127,8 @@ public void smallGraphWithWrongType() throws Exception {
 	}
 
 	@Test
+	@Ignore
```

Comment:
Actually, `@Ignore` is best used with a reason, which you give as a comment. Better:

```java
@Ignore("test program is not safe (external lacking output variables). This should throw some exception.")
```

The same applies to other occurrences below.

Reply:
Ah, I didn't know that.